styleguide/js/zoolander.js

```diff
@@ -139,6 +141,70 @@ jQuery(document).ready(function($) {
   };
 }(jQuery)); //to protect and scope the JQuery alias = $
 
+//Rackspace Zoolander Tabs jQuery Plugin
+(function($){
+  'use strict';
+  $.fn.rsTabs = function(){
+    var $tabs = $(this);
+    var $zoolanderWindow = $(window);
+
+    $tabs.each(function(){
+      var $thisTabSet = $(this);
+      var $tabLink = $thisTabSet.find('.rsTabs-menu .rsTabs-tabItem');
+
+      //function to reset the tabs on page resizes
+      var resizeTimer;
+      $zoolanderWindow.resize(function(){
+        clearTimeout(resizeTimer);
+        resizeTimer = setTimeout(resizeResetTabs, 250);
```

Comment:
I can switch this to 100ms also. Based on my experience it's not visually a noticeable difference though. I've tried this technique with 10 before and it still looks the same. Would be nice if we could see a diff in delay